modules/nvidia-gpu-virtualization.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * architecture/nvidia-gpu-architecture-overview.adoc
+
+:_content-type: CONCEPT
+[id="nvidia-gpu-virtualization_{context}"]
+= GPUs and virtualization
+
+Many developers and organizations are shifting to containerized applications and serverless infrastructure, but there is still huge interest in developing and maintaining applications running as virtual machines (VMs). Red Hat OpenShift Virtualization provides that ability and allows organizations to bring VMs into containerized workflows inside clusters.
+
+You can choose either of the following methods to attach the worker nodes to the GPUs:
+
+* GPU passthrough to access and use GPU hardware inside a virtual machine (VM).
+
+* GPU (vGPU) time slicing, when workloads do not saturate the GPU compute capacity.
```

Comment:
```suggestion
Many developers and enterprises are moving to containerized applications and serverless infrastructures, but there is still a lot of interest in developing and maintaining applications that run on virtual machines (VMs). Red Hat OpenShift Virtualization provides this capability, enabling enterprises to incorporate VMs into containerized workflows within clusters.

You can choose one of the following methods to connect worker nodes to GPUs:

* GPU passthrough to access and use GPU hardware within a virtual machine (VM).

* GPU (vGPU) time-slicing when GPU compute capacity is not saturated by workloads.
```

Reply:
s/ Red Hat OpenShift Virtualization / {VirtProductName} OR Red Hat {VirtProductName}

We're not consistent with our usage of time slicing. We have it written as both "time-slicing" and "time slicing"